test-requirements.txt

```diff
@@ -1,4 +1,6 @@
 pytest>=4.4.2
+pytest-cov==2.6.1
+codecov==2.0.15
```

Comment:
Can we try updating these two packages to the latest versions?

Reply:
Yes, updated package requirements.